orchagent/portsorch.cpp

```diff
@@ -5335,6 +5364,20 @@ bool PortsOrch::initializePort(Port &port)
         port.m_oper_status = SAI_PORT_OPER_STATUS_DOWN;
     }
 
+    // initalize port flap count
+    if (!flapCount.empty())
+    {
+        try
+        {
+            port.m_flap_count = to_uint<std::uint32_t>(flapCount);
+        }
+        catch (const std::exception &e)
+        {
+            SWSS_LOG_ERROR("Failed to get port (%s) flap_count: %s", port.m_alias.c_str(), e.what());
```

Comment:
@prgeor if we hit the exception, is it ok to still put the value to the DB?

Reply:
@nazariig fixed